modules/consul-cluster/variables.tf

```diff
@@ -164,6 +164,17 @@ variable "root_volume_delete_on_termination" {
   default     = true
 }
 
+variable "root_volume_encrypted" {
+  description = "Encrypt the volume at rest"
+  default     = false
+}
+
+variable "ebs_block_devices" {
+  description = "List of EBS volumes"
```

Comment:
Please update this description as follows:

1. Mention that this is the list of EBS volumes to create... But the user will have to attach and mount these volumes themselves: e.g., in a User Data script.
1. Specify that the type is a map from some unique name / ID for the volume (see comment below), which is used solely in this Terraform code, and that the values are objects with the fields supported by the `ebs_block_device` block in `aws_launch_configuration`. You may want to link to the `aws_launch_configuration` docs too.